Fix windows build

In [GitHub Actions runner](https://github.com/greynewell/musegpt/actions/runs/10864117849/job/30149040801), build is not working:

<img width="563" alt="Screenshot 2024-09-14 at 10 10 08 AM" src="https://github.com/user-attachments/assets/98151ce6-6888-4f18-bd3b-7be3b007aa99">

I would imagine Windows builds are not working across the board, but I don't have the hardware to test/fix.

Now that I added powershell scripts in #17, I can see the GitHub actions run is failing:
https://github.com/greynewell/musegpt/actions/runs/10896300308/job/30235880066?pr=16

We're getting the following error:
```
-- Finished setting up juceaide
CMake Warning at JUCE/extras/Build/CMake/FindWebView2.cmake:67 (message):
  WebView2 wasn't found in the the local NuGet folder.

  To install NuGet and the WebView2 package containing the statically linked
  library, open a PowerShell and issue the following commands

  > Register-PackageSource -provider NuGet -name nugetRepository -location
  https://www.nuget.org/api/v2

  > Install-Package Microsoft.Web.WebView2 -Scope CurrentUser
  -RequiredVersion 1.0.1901.177 -Source nugetRepository
-- Configuring incomplete, errors occurred!



  Alternatively you can use the JUCE_WEBVIEW2_PACKAGE_LOCATION CMake variable
  to specify the directory where this find script is looking for the
  *Microsoft.Web.WebView2* package directory.
Call Stack (most recent call first):
  JUCE/extras/Build/CMake/JUCEUtils.cmake:298 (find_package)
  JUCE/extras/Build/CMake/JUCEUtils.cmake:2066 (_juce_link_optional_libraries)
  JUCE/extras/Build/CMake/JUCEUtils.cmake:2111 (_juce_initialise_target)
  CMakeLists.txt:7 (juce_add_plugin)


CMake Error at C:/Program Files/CMake/share/cmake-3.30/Modules/FindPackageHandleStandardArgs.cmake:233 (message):
  Could NOT find WebView2 (missing: WebView2_include_dir WebView2_library)
Call Stack (most recent call first):
  C:/Program Files/CMake/share/cmake-3.30/Modules/FindPackageHandleStandardArgs.cmake:603 (_FPHSA_FAILURE_MESSAGE)
  JUCE/extras/Build/CMake/FindWebView2.cmake:80 (find_package_handle_standard_args)
  JUCE/extras/Build/CMake/JUCEUtils.cmake:298 (find_package)
  JUCE/extras/Build/CMake/JUCEUtils.cmake:2066 (_juce_link_optional_libraries)
  JUCE/extras/Build/CMake/JUCEUtils.cmake:2111 (_juce_initialise_target)
  CMakeLists.txt:7 (juce_add_plugin)


MSBuild version 17.11.2+c078802d4 for .NET Framework
MSBUILD : error MSB1009: Project file does not exist.
Switch: ALL_BUILD.vcxproj

Error: Process completed with exit code 1.
```

I remember something about this in the JUCE documentation, I'll look it up and link it here.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,7 +5,6 @@ add_subdirectory(JUCE)
 # Setup JUCE build settings, source code and compile definitions
 
 juce_add_plugin(musegpt
-    NEEDS_WEBVIEW2 TRUE
     NEEDS_WEB_BROWSER TRUE
     PLUGIN_MANUFACTURER_CODE Grey
     PLUGIN_CODE Muse 
@@ -24,7 +23,6 @@ target_compile_definitions(musegpt
         JUCE_DEBUG=1  # Enable debug mode
         JUCE_LOGGING=1  # Enable logging
         JUCE_STRICT_REFCOUNTEDPOINTER=1
-        JUCE_USE_WIN_WEBVIEW2_WITH_STATIC_LINKING=1
         JUCE_WEB_BROWSER=1
         JUCE_USE_CURL=0
         JUCE_VST3_CAN_REPLACE_VST2=0)

